### DISCUSSION TITLE
If a field has a note/description, show it in when hovering over the column header of tabular layout

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13614] [Feature Requests]
After looking at https://github.com/directus/directus/discussions/13277, I noticed that the admin-defined field note/description isn't shown in tabular layouts.

This could be easily added in the tabular layout index.ts file, though it may be wise to add a note in the collection admin as to where the field description is visible incase private system-related info is written.

This would clash for the string that is built for relational fields e.g., "relationPropertyName -> property", so maybe the field description tnat's shown when hovering over the grey dot, if it has one, could be suffixed on to this relational string?  e.g. `County -> Name` would become `County -> Name: The name of the county` (lame example!)

If a field description is supplied, maybe in the column header, instead of showing a dot (that's confusingly used to indicate a field is edited elsewhere), an info icon could be used.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
